### PR TITLE
Add trainees to a course

### PR DESCRIPTION
**Pull request by khanhkhanh**

### CODE REVIEW

File: app/controllers/supervisor/tasks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class TasksController < ApplicationController
+end
```
Comment (NguyenTanDuc) [inline app/controllers/supervisor/tasks_controller.rb]: new line

Comment (vjnhat) [inline app/controllers/supervisor/tasks_controller.rb]: insert a new line

File: app/views/supervisor/courses/_add_trainees.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<%= f.collection_check_boxes :user_ids, User.all, :id, :name do |b| %>
+  <%= b.label { b.check_box} %>
```
Comment (dieunb) [inline app/views/supervisor/courses/_add_trainees.html.erb]: coding convention

File: app/views/supervisor/courses/edit.html.erb [hunk 3]
```diff
@@ -1,7 +1,19 @@
...
+      <%= render 'form', f: f %>
+    <% end %>
+  </div>
+
+  <div class="col-md-6 col-md-offset-3" >
+    <br><br><br>
+    <h1>Add trainees to the course</h1>
+    <%= form_for [:supervisor, @course] do |f| %>
+      <%= render 'shared/error_messages', object: f.object %>
+	    <%= render 'add_trainees', f: f %>
```
Comment (dieunb) [inline app/views/supervisor/courses/edit.html.erb]: indent

File: app/views/supervisor/subjects/index.html.erb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<% provide :title, 'All Courses' %>
+<% provide :title, t(:all_courses) %>
 <h1> Subjects </h1>
 <%= will_paginate %>
 <li class="subject">
```
Comment (dieunb) [inline app/views/supervisor/subjects/index.html.erb]: ul???

File: db/seeds.rb [hunk 5]
```diff
@@ -10,10 +10,20 @@
              password_confirmation: "khanhkhanh",
              supervisor: false)
 
-50.times do |n|
-  Subject.create!(name: "#{n+1} name")
+48.times do |n|
+  name  = Faker::Name.name
+  email = "example-#{n+1}@tms.org"
+  password = "password"
+  User.create!(name:  name,
```
Comment (NguyenTanDuc) [inline db/seeds.rb]: space

**Pull request merged**